### PR TITLE
Adding week 11 blog post

### PR DESCRIPTION
**Pull request by xtanion**
blog visible in `blogs.html`

### CODE REVIEW

**Review by m-agour — COMMENTED** [review]
Hello @xtanion,
Nice post. I didn't find any typos, nor do I have any comments.

File: docs/source/posts/2022/2022-08-31-week-11-shivam.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Week 11 - Multiple transformations support and add tests 
+===========================================================
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-08-31-week-11-shivam.rst:2]: These lines are bigger than the actual title

File: docs/source/posts/2022/2022-08-31-week-11-shivam.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
+Week 11 - Multiple transformations support and add tests
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-08-31-week-11-shivam.rst:1]: Maybe `adding tests`

**Review by ganimtron-10 — COMMENTED** [review]
Nice Blog @xtanion ,
PTAL at the below comments.
Thanks!

**Comment by skoudoro** [comment]
LGTM, merging

**Pull request merged**